scripts/009_update_landiq.R

```diff
@@ -0,0 +1,127 @@
+library(tidyverse)
+
+load("~/ccmmf/LandIQ_data/crops_all_years.RData")
+
+dwr_2018 <- terra::vect(
```

Comment:
Since this is one-off data munging code, I will move it to a gist: https://gist.github.com/dlebauer/c7e16a0f8c741a85c8ff7794c002a558